docker/compose.yml

```diff
@@ -0,0 +1,16 @@
+name: adamant_example
+services:
+    adamant_example:
+        container_name: ${DOCKER_CONTAINER_NAME}
+        volumes:
+            - type: bind
+              source: ../../adamant
+              target: /home/user/adamant
+            - type: bind
+              source: ../../adamant_example
+              target: /home/user/adamant_example
+        extra_hosts:
+            - host.docker.internal:host-gateway
+        network_mode: host
+        image: ${DOCKER_IMAGE_NAME}
+        command: sleep infinity
```

Comment:
I think the command should actually run the `activate.sh` script to pre-install a lot of the dependencies for the user prior to login. however we only want this command to be run when we create the container... not when we start the container. I am not sure if this is what `command` does or not.

Reply:
I believe `command` in compose.yml can handle `activate.sh` - I'll test and push that change if this is the case.

Reply:
After looking around, we may need a bit more logic to only do this only the first time the container is run.

```
    command: >
      bash -c "if [ ! -f /initialized ]; then
                  # Run your one-time initialization command here
                  touch /initialized;
              fi;
```

but maybe you will find something better

Reply:
This works when following `docker compose up -d` with `docker compose exec` to run activation. Addressed in [221b3d0](https://github.com/lasp/adamant_example/pull/25/commits/221b3d0cef1c6b519b0848ad0246ab2ce72d27f7).